### PR TITLE
Fix a minor problem in colab 1_dqn_tutorial.ipynb file.

### PR DESCRIPTION
**Pull request by siavash-khodadadeh**
Call the collect steps function based on the hyperparameter "collect_steps_per_iteration".

### CODE REVIEW

File: tf_agents/colabs/1_dqn_tutorial.ipynb [hunk 1]
```diff
@@ -611,7 +611,8 @@
         "for _ in range(num_iterations):\n",
         "\n",
         "  # Collect one step using collect_policy and save to the replay buffer.\n",
-        "  collect_step(train_env, tf_agent.collect_policy)\n",
+        "  for col_step in range(collect_steps_per_iteration):\n",
```
Comment (the-unconscious-statistician) [inline tf_agents/colabs/1_dqn_tutorial.ipynb]: for col_steps in -> for _ in

File: tf_agents/colabs/1_dqn_tutorial.ipynb [hunk 2]
```diff
@@ -611,7 +611,8 @@
         "for _ in range(num_iterations):\n",
         "\n",
         "  # Collect one step using collect_policy and save to the replay buffer.\n",
```
Comment (kbanoop) [inline tf_agents/colabs/1_dqn_tutorial.ipynb]: collect a few steps

File: tf_agents/colabs/1_dqn_tutorial.ipynb (hunk 1 above)
Comment (kbanoop) [inline tf_agents/colabs/1_dqn_tutorial.ipynb]: for col_step in -> for _ in

**Comment by oars** [comment]
This is now merged. Not sure why copybara didn't close this. Investigating.

**Pull request closed without merging**